### ISSUE TITLE
Tagging task-force-specific discussions in email threads

### ISSUE THREAD

**Issue by Antoine-Zimmermann** [w3c/rdf-star-wg#44]
I propose that we add a prefix of the form [NameOfTaskForce] to the topic of a new email thread when it is specific to the task force called NameOfTaskForce. This issue is to get a unique proposal of what prefixes to use for each task force and enforce their use by the WG, so as to simplify filtering those discussions or finding them in the future. Here is a proposal:

- Use case task force: **[UC]** (clear, short, hardly possible to misspell it)
- Semantics task force: **[Sem]** (rather clear, short and hard to misspell)
- Editors task force: **[ED]** (somewhat clear, short and hardly possible to misspell)

Alternative:

- Use case TF: **[UC]**
- Semantics TF: **[Semantics]** (very clear but long and easy to misspell)
- Editors TF: **[Editors]** (very clear but possible to misspell)

**Comment by hartig** [comment]
+1 to the first option (i.e., [UC], [Sem], and [ED]).

**Comment by pchampin** [comment]
I don't think that this was ever adopted, but as we moved the largest part of the discussions to github issues, this does not really matter anymore.

@Antoine-Zimmermann any objection to close this issue?

**Comment by Antoine-Zimmermann** [comment]
Yes, do close it. Thanks.
